### ISSUE TITLE
Uncaught exception at TradfriDimmer.observe

### ISSUE THREAD

**Issue by dafes** [treban/pimatic-tradfri#10]
I receive the following Exception:
A Dimmer was configured but removed after i received the first exception. But even with no Dimmer configured i get the sam exception:

```
error [pimatic]: An uncaught exception occurred: TypeError: Cannot read property '0' of undefined
    at TradfriDimmer.observer (/home/pi/pimatic-app/node_modules/pimatic-tradfri/tradfri.coffee:365:32)
    at /home/pi/pimatic-app/node_modules/pimatic-tradfri/tradfri.coffee:2:1
    at ObserveReadStream.<anonymous> (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/tradfri-coapdtls.js:315:26)
    at emitOne (events.js:77:13)
    at ObserveReadStream.emit (events.js:169:7)
    at readableAddChunk (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/node_modules/readable-stream/lib/_stream_readable.js:198:18)
    at ObserveReadStream.Readable.push (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/node_modules/readable-stream/lib/_stream_readable.js:157:10)
    at ObserveReadStream.append (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/lib/observe_read_stream.js:32:10)
    at Agent.handle [as _handle] (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/lib/agent.js:298:27)
    at DtlsClientSocket.<anonymous> (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/lib/agent.js:95:14)
    at emitOne (events.js:77:13)
    at DtlsClientSocket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at DtlsClientSocket.Readable.push (_stream_readable.js:111:10)
    at DtlsClientSocket._onMessage (/home/pi/pimatic-app/node_modules/pimatic-tradfri/node_modules/tradfri-coapdtls/node_modules/node-mbed-dtls/client_socket.js:89:12)
    at emitTwo (events.js:87:13)
    at Socket.emit (events.js:172:7)
    at UDP.onMessage (dgram.js:480:8)
 This is most probably a bug in pimatic or in a module, please report it!
```

**Comment by treban** [comment]
I will check this.

**Comment by treban** [comment]
This bug should be fixed with version 0.1.13
